fabulaws/library/wsgiautoscale/servers.py

```diff
@@ -233,15 +233,15 @@ class DbMixin(PostgresMixin):
     """Mixin that creates a database based on the Fabric env."""
 
     # use the default version in this Ubuntu distro
-    postgresql_packages = ['postgresql', 'libpq-dev']
-    postgresql_tune = True
-    postgresql_shmmax = 107374182400 # 100 GB
-    postgresql_shmall = 26214400 # 100 GB / PAGE_SIZE (4096)
+    postgresql_packages = env.get('postgresql_packages', ['postgresql', 'libpq-dev'])
```

Comment:
Sure. Now that I dig into this a bit more, I also think that PostgresMixin would be a more appropriate place to process these, so I'll pass them on to there.